### ISSUE TITLE
problem with closing windows before opening new ones

### ISSUE THREAD

**Issue by shiroken01** [nicehash/NiceHashMiner#2525]
![screenshot](https://user-images.githubusercontent.com/20675389/137565297-b93759b3-440e-4e4a-a625-7c370b1031a5.png)

**Comment by S74nk0** [comment]
please send a log report and write the report ID here please 
![image](https://user-images.githubusercontent.com/1517233/137701851-cd0d51df-c861-41b6-a6c8-ee5abaedd57d.png)

**Comment by RomanMV** [comment]
Same problem
report ID: 7f25e404-9e4d-4f85-8024-20f09a1d4f88

**Comment by RomanMV** [comment]
Another computer has the same problem. Moreover, I found out that the problem manifests itself when the checkbox "Disable Device Status Monitor" is not checked. When it is not checked, the program is generally unrealistic to use on this computer, it is buggy all the time. It does not even copy the ID of the report, it crashes at the same time, I had to do it in the form of a screenshot. When this checkbox is checked, the program seems to work fine, it does not seem to crash 
screenshot report ID:  http://joxi.ru/12MwaP6iwG6evr
